steps/browserstack-bitrise-xcui-step/1.0.1/step.yml

```diff
@@ -0,0 +1,223 @@
+title: BrowserStack App Automate - XCUITest
+summary: |
+  Run your XCUITest tests on BrowserStack App Automate
+description: |
+  Run your XCUITest tests on BrowserStack App Automate. This step collects the built IPA from `$BITRISE_IPA_PATH`  and test suite from `$BITRISE_BUNDLE_PATH` Environment Variables
+website: https://github.com/browserstack/browserstack-bitrise-xcui-step
+source_code_url: https://github.com/browserstack/browserstack-bitrise-xcui-step
+support_url: https://github.com/browserstack/browserstack-bitrise-xcui-step/issues
+published_at: 2022-09-22T20:38:19.560375+05:30
+source:
+  git: https://github.com/browserstack/browserstack-bitrise-xcui-step.git
+  commit: 480c4bbdb8eb57b7eb5a995228d9b76d646ad0c2
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/browserstack/browserstack-bitrise-xcui-step
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- browserstack_username: null
+  opts:
+    description: Username of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Username for the BrowserStack account
+    title: BrowserStack username
+- browserstack_accesskey: null
+  opts:
+    description: Access Key of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Accesskey for the BrowserStack account
+    title: BrowserStack accesskey
+- app_ipa_path: $BITRISE_IPA_PATH
+  opts:
+    description: Path of the app (.ipa) file
+    is_expand: true
+    is_required: true
+    summary: Path to the app (.ipa) file
+    title: iOS app under test
+- opts:
+    description: Path of the output bundle (.app) file. Add either the $BITRISE_TEST_BUNDLE_PATH
```

Comment:
NOTE: descriptions are presented as a markdown content, `_` char should be escaped in `$BITRISE_TEST_BUNDLE_PATH`: -> `$BITRISE\_TEST\_BUNDLE\_PATH` or it should be formatted as a code: 
```
`$BITRISE_TEST_BUNDLE_PATH`
```